test/utils/conditions.go

```diff
@@ -19,7 +19,7 @@ package utils
 import (
 	"fmt"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
I would remove this file from the PR, you're fixing a pkg/ lint, I'm not sure why a test/ change is sneaking in